.changeset/happy-books-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-home': minor
```

Comment:
Think that this should be a patch change right? it's not breaking and `plugin-home` is on `0.x`?

Reply:
Thanks @benjdlambert for pointing it out 🙏🏻 
@soapraj asked me to update the changeset on behalf of @aitherios.
Here is the commit 🙌🏻: https://github.com/backstage/backstage/pull/20020/commits/d65cb238e8a661c05a11f9c66740e419a8d61384